### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by LlmDl**

### CODE REVIEW

File: resources/lang/de-DE.yml [hunk 1]
```diff
@@ -715,6 +715,7 @@ msg_error_ta_plot_claim: 'Der Spielername fehlt. Bsp: /ta plot claim PlayerNameH
 msg_error_no_player_with_that_name: 'Es konnte kein Spiel Spieler mit dem Namen %s gefunden werden.'
 msg_admin_has_given_you_a_plot: 'Ein Admin hat für dich einen Plot bei %s beansprucht.'
 msg_town_forcepvp_setting_set_to: 'Die PvP-Einstellung der Stadt %s wurde geändert zu %s'
+msg_town_forcedisabledpvp_setting_set_to: 'The town %s has had their admindisabledpvp setting forced to %s'
```
Comment (nicosammito) [inline resources/lang/de-DE.yml:718]:
```suggestion
msg_town_forcedisabledpvp_setting_set_to: 'Die gezwungene PVP-Einstellung der Stadt %s wurde geändert zu %s'
```

File: resources/lang/de-DE.yml [hunk 2]
```diff
@@ -1573,3 +1574,5 @@ msg_err_cannot_add_nation_over_resident_limit: '&cYour nation has reached the ma
 msg_err_cannot_join_nation_over_resident_limit: '&cYou cannot join this nation as it would be over the residents-per-nation limit of %s.'
 #Message shown to a town because the nation has hit their max-towns-limit.
 msg_err_cannot_join_nation_over_town_limit: '&cYou cannot join this nation, it has reached the maximum number of allowed towns-per-nation: %s.'
+#Message shown confirming /tw set usedefaults
+confirmation_are_you_sure_you_want_to_reset_this_worlds_settings: 'Are you sure you want to reset all of this world''s settings to the ones which are set in the config?'
```
Comment (nicosammito) [inline resources/lang/de-DE.yml:1578]:
```suggestion
confirmation_are_you_sure_you_want_to_reset_this_worlds_settings: 'Sind Sie sicher, dass Sie alle Einstellungen dieser Welt auf die Einstellungen in der Konfiguration zurücksetzen wollen?'
```

**Review by nicosammito — COMMENTED** [review]
fixed some German translation

File: resources/lang/de-DE.yml (hunk 1 above)
Comment (LlmDl) [reply resources/lang/de-DE.yml:718]: Translations are to be done on crowdin @nicosammito

**Pull request merged**